src/benchq/data_structures/resource_info.py

```diff
@@ -13,9 +13,9 @@
 class DecoderInfo:
     """Information relating the deceoder."""
 
-    total_energy_consumption: float
-    power: float
-    area: float
+    total_energy_consumption_in_nanojoules: float
+    power_in_nanowatts: float
+    area_in_micrometers: float
```

Comment:
`area_in_micrometers_squared` . I think Joe mentioned this in an email a few days before the report deadline

Reply:
Done!